class/Defaults/MergeTag/Avatar.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Avatar merge tag class
+ *
+ * Requirements:
+ * - Trigger property `user_object` or any other passed as
+ * `property_name` parameter. Must be an object, preferabely WP_User
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\MergeTag;
```

Comment:
I'd like this merge tag to sit under `User` namespace, so it would be:

```suggestion
namespace BracketSpace\Notification\Defaults\MergeTag\User;
```

The reason why you are getting the fatal error is that the class name is `AvatarTag` and file name is `Avatar`. It must match

Reply:
Thanks, I'll get things moved around and push shortly.